cmd/ebrelayer/main.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+// -------------------------------------------------------------
+//      Main (ebrelayer)
+//
+//      Implements CLI commands for the Relayer service, such as
+//      initalization and event relay.
+// -------------------------------------------------------------
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/rpc"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/libs/cli"
+
+	// "golang.org/x/crypto"
```

Comment:
can you remove this or add a TODO or comment to why is this line commented?

Reply:
fixing in next push

Reply:
https://github.com/cosmos/peggy/commit/969d3035f2c82a46f07bfd4ec5cd61d75d9a131a, https://github.com/cosmos/peggy/commit/39543e454fa623483134ca716c4e5f7e1c674534

Reply:
https://github.com/cosmos/peggy/commit/969d3035f2c82a46f07bfd4ec5cd61d75d9a131a, https://github.com/cosmos/peggy/commit/39543e454fa623483134ca716c4e5f7e1c674534